en/docs/learn/cors-migration.md

```diff
@@ -0,0 +1,515 @@
+# CORS Migration
+
+Complete the following steps for the CORS migration.
+
+1. Remove any CORS configurations defined at `repository/resources/conf/templates/repository/conf/tomcat/web.xml.j2`. 
+You will be able to do this by removing the whole tag under the filter class named `com.thetransactioncompany.cors.CORSFilter`. If 
```

Comment:
Why do we need to edit .j2 files. Can't we remove them by-default from .j2 files. Because originally, we don't advice to edit .j2 file. How can someone coming from 5.9.0 have added any configs to .j2 file. Have we mentioned them any 5.9.0/5.10.0 document?

Reply:
There are no CORS configurations by default in .j2 files. The CORS instructions so far instruct adding configurations to the .j2 files. Therefore in migration, it is necessary to remove the CORS configurations only if they were added.

Reply:
As I have checked there is no reference to add this configuration to .j2 file. So ideally this should not be there. So having a step to remove this configuration is not required